converter/src/test/java/gov/cms/qpp/conversion/model/error/DetailTest.java

```diff
@@ -9,23 +8,6 @@
 import static com.google.common.truth.Truth.assertWithMessage;
 
 public class DetailTest {
-	@Test
-	public void stringRepresentation() {
```

Comment:
Will we loose code coverage due to the removal of these tests?

Reply:
no